docs/fast_data/faq/bootstrapping-fast-data.md

```diff
@@ -0,0 +1,154 @@
+
+You can also specify how many replicas are needed to send acknowledgments on produced messages before committing, to guarantee the synchronization of the distributed data.
+
+This value can be set at the topic level with the parameter `min.insync.replicas` and is usually equal to `# of brokers - 1`. Critical flows may require a number of in-sync replicas equal to the number of brokers. 
+
+## Mongo Db
+
+### What should be the required resources to perform an Initial Load?
+
+The **Initial Load** is a manual operation made at runtime to initialize the projection's records with the most recent copy of the dataset from the system of record linked to it.
+
+This operation is carried out when we need the more recent copy of a table, in order to transfer the data present within the Fast Data system, with a **Change Data Capture** (**CDC**) and will be then processed by the [Real-Time Updater (RTU)](/fast_data/realtime_updater.md).
+
+:::danger Before executing the Initial Load
+Since an high number of ingestion messages will be handled by the RTU, remember **to disable strategy** during the execution of the Initial Load.
+
+This will avoid to perform additional MongoDb queries made by the strategies to generate projection changes records.
+:::
+
+:::tip Execution Time
+An initial load of `6.000.000 records` took around _10 minutes_, which is approximately a rate of `60k message/minute`: for this message rate, up until 70% of CPU from a single MongoDb core was used.  
+:::
+
+### What should be the required resources to perform a Single View Refresh?
+
+The **Single View Refresh** is an operation performed to update all the projection changes of a single view: this activity is needed during the first release of a single view and every time a new feature is released in the environment, since it will start the aggregation pipeline of the [**Single-View Creator**. (**SVC**)](/fast_data/single_view_creator.md)
+
+On a [Standard Architecture](/fast_data/architecture.md#standard-architecture), this means that the database will perform operations on:
+
+* the projection changes collections, because the **SVC** will update the projection changes record updating/removing the `changes` record; 
+* the projections that are used by the single view aggregation mechanism; 
+* the single view collections, to insert/update/delete records corresponding to single views.
+
+:::tip Execution Time
+The time generation is **strictly dependent** on the number of projections that are needed to compute all the fields of a single view. 
+
+On a single view having three projections and custom functions, a refresh operation involving 16k projection changes records required _5 minutes_, which is approximately a rate of `1k single view/minute`, using up until the 80% of a single MongoDb core.
```

Comment:
Consindering MongoDB perfs, I believe we should keep track also of the *max Disk IOPS* (at least make the instance setting explicit)

Reply:
hi @danibix95 , i don't have this information available